python/python/res/job_queue/job_manager.py

```diff
@@ -336,6 +336,7 @@ def postMessage(self, job=None, extra_fields={}, url=None):
 
         payload = {"user": self.user,
                    "cwd": os.getcwd(),
+                   "application": "ert",
```

Comment:
It is not the same process, but is it not the same application? Or "system" maybe? It is definitely the same something, and I think this 'something' is what we want to capture with 'application'. 
When I go looking at my Ert-logs is it not natural that I see the logs for both the main ert process and the forward models? 

But I also agree that there is definitely a difference between the main ert application and the forward models which should be explicit in the log-messages. One option is to add an ert-specific keyword like "ert_context" (or similar), and another option (which I am slightly partial to) is to add a more general keyword like "subprocess" / "subsystem" / "service" which in this case can be "ert_forward_model", and for other applications whatever is natural for them (including empty).